cpp/src/aztec/honk/sumcheck/sumcheck_round.test.cpp

```diff
@@ -100,13 +101,15 @@ TEST(SumcheckRound, ComputeUnivariateVerifier)
     FF id_3 = { 0 };
     FF lagrange_first = { 0 };
     FF lagrange_last = { 0 };
+    FF pow_zeta = { 1 };
 
     // q_m * w_l * w_r + q_l * w_l + q_r * w_r + q_o * w_o + q_c
```

Comment:
Add pow_zeta to this comment? Seems to me we should be using the real expression for this test anyway rather than a hard coded `54`..

Reply:
Yeah, we should generate a full, nontrivial example and use that in various places.